src/py_tortoize/scan_and_collect_results.py

```
@@ -0,0 +1,189 @@
+    residues = flatten_residues(result)
+    residues["path"] = path
+
+    protein_level_stats = get_protein_level_z_scores(result)
+    protein_level_stats["path"] = path
+    return protein_level_stats, residues
+
+
+def main(parent_directory, output_file_prefix, target_file_pattern):
+    paths = crawl_dir_by_depth(parent_directory, target_file_pattern, 5)
+    results = Parallel(n_jobs=16)(delayed(get_stats_for_single_path)(path) for path in paths)
+    all_residue_results, all_protein_results = tuple(zip(*results))
```

Ruff (0.14.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)